src/uwtools/resources/jsonschema/ww3.jsonschema

```diff
@@ -0,0 +1,44 @@
+{
+  "properties": {
+    "ww3": {
+      "additionalProperties": false,
+      "properties": {
+        "namelist": {
+          "additionalProperties": false,
+          "anyOf": [
+            {
+              "required": [
+                "base_file"
+              ]
+            },
+            {
+              "required": [
+                "update_values"
+              ]
+            }
+          ],
```

Comment:
In re: template render vs config realize, probably these are both required: We need the `base_file` that contains the template, and we need the `update_values` to render the template. I do like keeping these names, though, so that we won't have to change terminology later if we implement a real namelist config realize. So, I think this `anyOf` can be replaced with a
```
"required": ["base_file", "update_values"],
```

Reply:
I think it still bothers me requiring an `update_values` where the expected behavior is to provide a single base file and not change it. I'm not sure theres a better way at this stage though

Reply:
I thought we were going to support Jinja2-templated files and render those. @christinaholtNOAA what do you recall about this?

Reply:
Discussed offline - we can probably make `update_values` optional with the expectation of a user providing a complete base file that can be overridden